spec/cirro_io_v2/resources/user_spec.rb

```diff
@@ -20,6 +20,35 @@
     }
   end
 
+  describe '#create' do
+    let(:params) do
+      {
+        "object": 'user',
+        "first_name": 'Maynard',
+        "last_name": 'Keenan',
+        "time_zone": 'Berlin',
+        "birthday": '1975-11-22',
+        "country_code": 'DE',
```

Comment:
🤔 
not sure this is enough. we need to try this version of the gem with `engineering-space` or `app` or `automate

Reply:
nevermind. now I see second PR

Reply:
in Cirro users are created nimbus-style via DataProducer::User, meaning everything is seeded except anything that's provided. We can skip any params whatsoever and create a valid user.